kornia/models/__init__.py

```diff
@@ -1 +1,3 @@
 from . import detector
+from . import segmentor
```

Comment:
I’d call the modules defining they category name
- detection
- segmentation 
- tracking

in this way seems that there’s a single model per category?